pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: pouch
 description: Generic Dart utilities and helper functions
-version: 0.7.0
+version: 0.8.0
 homepage: https://github.com/jesims/pouch
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.15.0 <3.0.0'
```

Comment:
2.15 Enables Constructor tear-offs

Reply:
Bumping the base SDK environment is considered a breaking change in semver. It's like going from Java 8 to Java 9.

Reply:
But it's a minor version....

Reply:
Breaking change it is. Updated changelog and verion